File: src/aws/lambdas/incremental_distribution/src/main/java/uk/nhs/nhsx/core/auth/ApiName.java
package uk.nhs.nhsx.core.auth;

// maps to api name defined in -> tools/build/lib/nhsx/helpers/target.rb
public enum ApiName {
    Mobile("mobile"),
    Health("health"),
    HighRiskPostCodeUpload("highRiskPostCodeUpload"),
    TestResultUpload("testResultUpload"),
    HighRiskVenuesUpload("highRiskVenuesCodeUpload"),
    IsolationPayment("isolationPayment");

    public final String name;

    ApiName(String name) {
        this.name = name;
    }
}

File: src/aws/lambdas/incremental_distribution/infra/src/main/kotlin/uk/nhs/nhsx/core/auth/ApiName.kt
package uk.nhs.nhsx.core.auth

// maps to api name defined in -> tools/build/lib/nhsx/helpers/target.rb
enum class ApiName(val keyName: String) {
    Mobile("mobile"),
    Health("health"),
    HighRiskPostCodeUpload("highRiskPostCodeUpload"),
    TestResultUpload("testResultUpload"),
    HighRiskVenuesUpload("highRiskVenuesCodeUpload"),
    IsolationPayment("isolationPayment");
}
